samples/drivers/video/capture_to_lvgl/README.rst

```diff
@@ -0,0 +1,74 @@
+.. zephyr:code-sample:: video-capture-to-lvgl
+   :name: Video capture to LVGL
+   :relevant-api: video_interface
+
+   This sample application demonstrates how to use the video API to retrieve video
+   frames from a capture device and display them on an LCD via the LVGL library.
+
+Description
+***********
+
+The application uses the :ref:`Video API <video_api>` to retrieve video frames from
+a video capture device, writes a frame count message to the console, and then sends
+the frame to an LCD display.
+
+Requirements
+************
+
+This sample requires a supported :ref:`video capture device <video_api>` (e.g., a camera)
+and a :ref:`display <display_api>`.
```

Comment:
This implies that the `sw_generator` is a capture device... which is true to some extent!